.github/workflows/dcm-label.yaml

```diff
@@ -0,0 +1,20 @@
+# A CI workflow to remind the contributor to add the run-dcm-workflow label.
+
+name: DCM Label Reminder
+
+on:
+  pull_request:
+    types: [synchronize, opened, reopened, labeled, unlabeled]
+    paths:
+      - '**/*.dart'
+
+jobs:
+  changelog-reminder:
+    if: ${{ !contains(github.event.*.labels.*.name, 'run-dcm-workflow') }}
+    name: Prevent submission
+    runs-on: ubuntu-latest
+    steps:
+      - name: Label reminder
+        run: |
+          echo "Please add the 'run-dcm-workflow' label to run DCM checks."
```

Comment:
You can have this pop up as an error in the UI using ::error
https://docs.github.com/en/actions/using-workflows/workflow-commands-for-github-actions#using-workflow-commands-to-access-toolkit-functions